Resources/Prototypes/Entities/Clothing/Back/satchel.yml

```diff
@@ -178,3 +178,22 @@
     maxItemSize: Huge
     grid:
     - 0,0,19,9
+
+- type: entity
+  parent: [ClothingBackpackSatchel, BaseSyndicateContraband]
+  id: ClothingSatchelSmugglers
+  name: Smugglers Satchel
+  description: A backpack so slim it can fit under the tiles of the station
+  components:
+  - type: Storage
+    maxItemSize: Large
+    grid:
+    - 0,0,1,3
+    - 3,0,6,3
+    - 8,0,9,3
+  - type: Sprite
+    sprite: Clothing/Back/Satchels/smuggler.rsi
+    visible: false
+  - type: SubFloorHide
+  - type: Anchorable
+  - type: Appearance
```

Comment:
This entity should be introduced in `Resources/Prototypes/_Harmony/Entities/Clothing/Back/satchel.yml` instead.

Reply:
Additionally, `name` should be lowercase to match every other item name in the game.